### DISCUSSION TITLE
Docs: Error when using Resender

### DISCUSSION THREAD

**Discussion by miguelvelasquezdev** [resend/react-email#710] [Q&A]
### Describe the Bug

Docs say that we should write the code like that:

<img width="813" alt="Screenshot 2023-05-18 at 5 42 05 PM" src="https://github.com/resendlabs/react-email/assets/59978150/84e31e3e-4ffd-47a3-a38c-0a7e4a0cbddc">

but when I use this code, i get this error:
<img width="738" alt="Screenshot 2023-05-18 at 5 54 47 PM" src="https://github.com/resendlabs/react-email/assets/59978150/c326452e-e6b1-4d12-a6df-48222544a616">

[Resend docs](https://resend.com/docs/send-with-nextjs) says that it should be like that:

<img width="817" alt="Screenshot 2023-05-18 at 5 57 35 PM" src="https://github.com/resendlabs/react-email/assets/59978150/ddc2afa5-8770-4075-b62b-257d834532e1">

(Sorry if it's a false alarm, and maybe I'm using it in a wrong way.)

### Which package is affected (leave empty if unsure)

docs

### Link to the code that reproduces this issue

https://github.com/resendlabs/react-email/blob/d158ad77e3798808b4efbd322b618922da12529e/docs/integrations/resend.mdx?plain=1#LL68C53-L68C53

### To Reproduce

```ts
import { Resend } from 'resend';
import { Email } from './email';

const resend = new Resend('re_123456789');

resend.sendEmail({
  from: 'you@example.com',
  to: 'user@gmail.com',
  subject: 'hello world',
  react: <Email firstName="John" product="MyApp" />,
});
```

### Expected Behavior

I expect to don't get this error:

'EmailReceipt' refers to a value, but is being used as a type here. Did you mean 'typeof EmailReceipt'?ts(2749)

### What's your node version? (if relevant)

18.15.0

**Comment by rfoel** [comment]
It seems this error is caused because the extension of your file is not `.jsx` nor `.tsx`.

**Comment by bukinoshita** [comment]
Are you using with Nextjs? If not, you will need a transpiler for node.js to understand jsx

**Comment by MahmoudTahaCaisy** [comment]
@miguelvelasquezdev 

Are you find the solution?
